pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandalone.java

```diff
@@ -375,7 +375,7 @@ private void createSampleNameSpace(ClusterData clusterData, String cluster) {
             }
 
             // Create marker for "global" cluster
-            if (!admin.clusters().getClusters().contains(globalCluster)) {
+            if (admin.clusters().getCluster(globalCluster) == null) {
```

Comment:
I think this would throw exception instead of returning a null, if the cluster is not there.

Reply:
Thanks merlimat